src/Microsoft.OData.Client/Materialization/CollectionValueMaterializationPolicy.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.OData.Client.Materialization
     using Microsoft.OData;
     using Microsoft.OData.Edm;
     using DSClient = Microsoft.OData.Client;
+    using System.Collections.ObjectModel;

```

Comment:
Nit:This should probably be somewhere on line 12...

Reply:
Fixed